myning/chapters/enter_mine.py

```diff
@@ -214,7 +216,12 @@ def add_trip_to_player():
             )
             pick(["Awesome!"], message=message, sub_title=subtitle)
 
-    player.add_xp(trip.experience_gained)
+    if len(player.army) > 1:
+        xp = int(trip.experience_gained * 1 / 2 * len(player.army) * macguffin.xp_boost)
```

Comment:
is this just a huge buff to the amount of xp gained from a mine? Having 2 or more members in the army just is a profit in xp compared to before. so it becomes "meta" to have as many people as possible to get the maximum xp per trip. This does bode well for the current scaling of xp and how difficult it is to level up past 20-30s

Reply:
This is the exact code that was already in the player method I'm removing, so there's no buff. But yes, the meta is to have a ton of people so you get the scaling xp bonus.